src/room_data.c

```diff
@@ -1064,7 +1064,7 @@ TbBool rectreate_repositioned_body_in_room_on_subtile(struct Room *room, MapSubt
             struct Thing* bodytng = create_dead_creature(&pos, rrepos->models[ri], 0, room->owner, rrepos->explevels[ri]);
             if (!thing_is_invalid(bodytng))
             {
-                bodytng->byte_14 = 1;
+                bodytng->byte_14 = 1; // Laid to rest
```

Comment:
It would be good to add a new 
```
struct {
unsigned char byte_13c;
unsigned char laid_to_rest;
} corpse;
```
to that big union inside `struct Thing` at `thing_data.h`

Reply:
Like this?